docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18903,6 +18903,555 @@
+              }
+            },
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.MachineLearningServices/workspaces"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/allowPublicAccessWhenBehindVnet",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/allowPublicAccessWhenBehindVnet",
+                        "notEquals": false
+                      }
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-MachineLearning-PublicAccessWhenBehindVnet"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-MachineLearningAks",
+            "mode": "Indexed",
+            "description": "Deny AKS creation (not attaching) in machine learning.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Machine Learning"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "Audit",
+                  "Disabled",
+                  "Deny"
+                ],
+                "defaultValue": "Deny"
+              }
+            },
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.MachineLearningServices/workspaces/computes"
+                  },
+                  {
+                    "field": "Microsoft.MachineLearningServices/workspaces/computes/computeType",
+                    "equals": "AKS"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.MachineLearningServices/workspaces/computes/resourceId",
+                        "exists": false
+                      },
+                      {
+                        "value": "[[empty(field('Microsoft.MachineLearningServices/workspaces/computes/resourceId'))]",
+                        "equals": true
+                      }
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-MachineLearningAks"
```

Comment:
can you change to Deny-MachineLearning-Aks?

Reply:
DONE.